src/ethereum/arrow_glacier/transactions.py

```diff
@@ -43,6 +43,16 @@ class LegacyTransaction:
     r: U256
     s: U256
 
+@slotted_freezable
+@dataclass
+class Access:
+    """
+    Hoisted from AccessListTransaction for readability
```

Comment:
Instead of documenting where this type came from, document what it is. Maybe:

```suggestion
    A mapping from account address to storage slots that are pre-warmed as part of a transaction.
```